core/parties/dependencies.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2023 Gobalsky Labs Limited
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package parties
+
+import (
+	"code.vegaprotocol.io/vega/core/events"
+)
+
+//go:generate go run github.com/golang/mock/mockgen -destination mocks/mocks.go -package mocks code.vegaprotocol.io/vega/core/parties Broker
```

Comment:
The broker package is the one exception to the rule of _"declare dependency interfaces and generate mocks in the package"_. Keeping the interface here is fine, but just import `core/broker/mocks`

Reply:
Ok